### PR TITLE
Arrange dependencies

### PR DESCRIPTION
**Pull request by negokaz**
## TODO

- [x] Remove `akka-slf4j` dependency (in Compile configuration)
- [x] Remove `akka-persistence-cassandra` dependency
- [x] ~~Remove `akka-cluster-typed` dependency~~
  - Akka Projection requires actor-typed that [requires akka-cluster-typed when the cluster is enabled](https://github.com/akka/akka/blob/v2.6.9/akka-actor-typed/src/main/scala/akka/actor/typed/internal/receptionist/ReceptionistImpl.scala#L21-L32)
- [x] Resolve warning in sbt configuration
- [x] Modify guides

### CODE REVIEW

**Review by tksugimoto — DISMISSED** [review]

File: docs/implementation_guide.md [hunk 1]
```diff
@@ -329,89 +288,45 @@ lerna.akka.entityreplication.raft.eventhandler {
...
+    journal.plugin  = ""
+    query.plugin    = ""
 }
 ```
 
-## Cassandra driver configuration
+It requires explicit user configuration by overriding them in the application.conf.
 
-akka-entity-replication has default profile settings for DataStax Java Driver.
+For an example configuration to use Cassandra as a data store with [akka-persistence-cassandra](https://doc.akka.io/docs/akka-persistence-cassandra/current/) see [akka-entity-replication-with-cassandra.conf](/src/main/resources/akka-entity-replication-with-cassandra.conf).
```
Comment (xirc) [inline docs/implementation_guide.md:320]: IMHO, could we move `akka-entity-replication-with-cassandra.conf` from `/src/main/resources` to `/src/test/resources`?
`/src/main/resources/akka-entity-replication-with-cassandra.conf` may be assembled into published jar file even if it is not necessary for library users.

If you have any reason to put the configuration file into `/src/main/resources`, could you let me know why?

**Review by xirc — COMMENTED** [review]
Great! Thank you!

File: docs/implementation_guide.md (hunk 1 above)
Comment (negokaz) [reply docs/implementation_guide.md:320]: @xirc 

I have no reason for it. I will move `akka-entity-replication-with-cassandra.conf` to `/src/test/resources`.

Comment (negokaz) [reply docs/implementation_guide.md]: @xirc I moved the file.
b52327a

**Review by xirc — APPROVED** [review]
LGTM

**Pull request merged**